client/common/directives/ModelField/ModelField.js

```diff
@@ -222,7 +222,7 @@ angular.module('dashboard.directives.ModelField', [
       case 'POI':
         template = '<label class="col-sm-2 control-label">{{ display.label || key }}:</label>\
           <div class="col-sm-10">\
-            <model-field-points-of-interest-edit key="key" property="property" options="display.options" ng-model="data[key]" class="field" ng-required="{{ model.properties[key].required }}" disabled="display.readonly"></model-field-points-of-interest-edit>\
```

Comment:
@msirrele what was the reason for having disabled="display.readonly" in the first place?